src-java/base-topology/base-storm-topology/src/release/resources/topology.properties.example

```diff
@@ -102,6 +102,7 @@ floodlight.alive.timeout = 10
 floodlight.alive.interval = 2
 message.blacklist.timeout = 180
 floodlight.regions = 1
+# floodlight.switch.mapping.remove.delay.seconds = 900
```

Comment:
why is this commented, not removed?

Reply:
See response for this option into template file.